.github/workflows/clojure.yml

```diff
@@ -40,11 +40,23 @@ jobs:
     - name: Run midje tests
       run: bb test:midje
 
-    - name: Run clj tests
-      run: bb test:clj
+    - name: Run clj tests with Clojure 1.8
+      run: bb test:clj -A:1.8
 
```

Comment:
So in bb.edn we have:

`(apply clojure "-A:dev:clj-test:test-runner" *command-line-args*)`

Which means we can specify `-A` multiple times? I did not know/remember that.

A bit pedantic maybe but: `-M` has replaced `-A`. So probably should be using `-M` instead of `-A`.

Reply:
hmmm, yeah, I see those CLI warnings about `-A` being deprecated.
I tried replacing `-A` with `-M` in `clj:test`'s `(apply clojure "-A:dev:clj-test:test-runner" *command-line-args*)` and it makes it so passing an additional `-A:1.9` doesn't work :/

Reply:
So if my wee bit of experiment at the terminal is right, it seems multiple `-A`s concatenate whereas for multiple `-M`s the first one wins. 

Your repeating `-A` trick seems to keep the code nice and simple at the cost of a warning and (perhaps) taking advantage of an obscure behaviour.

Much less verbose than the little wrappers I tend to write. [Here's one as an example if you are interested](https://github.com/clj-commons/clj-yaml/blob/4bd5bdf1c068cec31ba96e3089f34593f59b7875/script/test_clj.clj).